<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpiritInfoDialog</class>
 <widget class="QDialog" name="SpiritInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>731</width>
     <height>561</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="bag_2">
    <attribute name="title">
     <string>背包</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>330</y>
       <width>371</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>技能：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="skill_name_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>技能</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>技能效果：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="skill_effect_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>技能效果</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>251</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>类别：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="type">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>类别</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>进化等级</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="evolution">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>进化等级</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>等级：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="level">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>等级</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>经验：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="exp">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>经验</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>血量：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hp">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>血量</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>攻击力：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="attack">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>攻击力</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>防御力：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="defense">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>防御力</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>攻击间隔：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="attack_interval">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>攻击间隔</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="last_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>470</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>上一只</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>470</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>下一只</string>
     </property>
    </widget>
    <widget class="QPushButton" name="all_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>所有</string>
     </property>
    </widget>
    <widget class="QLabel" name="bag_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 190, 186);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pic">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>321</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>bag_back</zorder>
    <zorder>layoutWidget_3</zorder>
    <zorder>layoutWidget_4</zorder>
    <zorder>last_2</zorder>
    <zorder>next_2</zorder>
    <zorder>all_2</zorder>
    <zorder>pic</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
